yt/python/yt/wrapper/schema/types.py

```diff
@@ -171,34 +200,33 @@ def _get_time_types():
 
 
 def _get_py_time_types():
-    if hasattr(_get_py_time_types, "_info"):
-        return _get_py_time_types._info
-    check_schema_module_available()
```

Comment:
Maybe I've recklessly deleted this code, but it looks like it doesn't depend on python's version and skiff_bindings.

Reply:
Yes, it's common behavior, but that looks kinda strange.

I expected `check_schema_module_available` check in the top-level functions/objects, who really need skiff to work.

But I agree with you, it's not a good idea to change this only in one function.